### PR TITLE
circleci config file update to build and test challenges

### PR DESCRIPTION
**Pull request by Z-a-r-a-k-i**

### CODE REVIEW

**Comment by Z-a-r-a-k-i** [comment]
Need help on this, trying to make circleci build every challenge (docker-compose) into challenges folder.

File: .circleci/config.yml [hunk 1]
```diff
@@ -49,6 +49,15 @@ jobs:
           git diff -w | cat
           git diff-index -w --quiet HEAD --
 
+  challenge-build:
+    working_directory: /go/src/pathwar.land
+    machine:
```
Comment (moul) [inline .circleci/config.yml]:
```suggestion
    docker:
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -49,6 +49,15 @@ jobs:
...
+      image: docker:18.06.3-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: make build
```
Comment (moul) [inline .circleci/config.yml]:
```suggestion
      - run: cd challenges && make build
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -49,6 +49,15 @@ jobs:
           git diff -w | cat
           git diff-index -w --quiet HEAD --
 
+  challenge-build:
+    working_directory: /go/src/pathwar.land
+    docker:
+      image: docker:18.06.3-ce-git
```
Comment (moul) [inline .circleci/config.yml]:
```suggestion
      - image: docker:18.06.3-ce-git
```

**Comment by Z-a-r-a-k-i** [comment]
Thanks !

**Pull request merged**